ui/src/app/records/search/brief-view/libraries-brief-view.component.ts

```diff
@@ -93,11 +93,11 @@ import { ToastrService } from 'ngx-toastr';
     <div id="collapseBasic" [collapse]="isCollapsed">
       <ul *ngIf="locations.length" class="list-group list-group-flush">
         <li *ngFor="let location of locations;" class="list-group-item p-1">{{ location.metadata.name }}
-          <a (click)="deleteLocation(location.metadata.pid)"
```

Comment:
Do not forget to do the same in the new `rero-ils-ui` project.

Reply:
will create a new PR in the `rero-ils-ui` project